pkg/scheduler/core/generic_scheduler.go

```diff
@@ -150,7 +150,8 @@ func (g *genericScheduler) snapshot() error {
 // If it succeeds, it will return the name of the node.
 // If it fails, it will return a FitError error with reasons.
 func (g *genericScheduler) Schedule(ctx context.Context, prof *profile.Profile, state *framework.CycleState, pod *v1.Pod) (result ScheduleResult, err error) {
-	trace := utiltrace.New("Scheduling", utiltrace.Field{Key: "namespace", Value: pod.Namespace}, utiltrace.Field{Key: "name", Value: pod.Name})
+	//trace := utiltrace.New("Scheduling", utiltrace.Field{Key: "namespace", Value: pod.Namespace}, utiltrace.Field{Key: "name", Value: pod.Name})
```

Comment:
should remove this old trace code.

Reply:
The commented out trace is not old. It is 1.18 version of trace with utiltrace being used. Arktos hasn't adopted this change. Not sure whether we want to adopt it until we can have perf data to support it.